tfutils/model.py

```diff
@@ -196,7 +196,7 @@ def dropout(self, dropout=.5, in_layer=None):
 
 def mnist(inputs, train=True, **kwargs):
     m = ConvNet(**kwargs)
-    reuse = None if train else True
+    reuse = None #if train else True
```

Comment:
I came across this issue as well. One way to deal with it is to explicitly pass reuse kwargs. On the other hand, this is dealt with in the `get_valid_targets_dict`, so maybe we no longer need to set reuse here.